bossdata/remote.py

```diff
@@ -197,14 +200,30 @@ def get(self, remote_path, progress_min_size=10, auto_download=True):
                 then a RuntimeError occurs.
 
         Returns:
-            str: Absolute local path of the local file that mirrors the remote file.
+            str, int: Absolute local path of the local file that mirrors the remote file, and
```

Comment:
I just realized why my suggested pattern for detecting substitutions by get() doesn't work (because remote_paths[0] isn't a local path!) but I just update the get() logic to fix this in cf610b7de487f77834.